### PR TITLE
Add 'output' mode to grimshot

### PR DESCRIPTION
**Pull request by cyrinux**
Hi,

This adds the possibility to capture only the focused screen(s) if several are used

Regards,

### CODE REVIEW

File: contrib/grimshot [hunk 1]
```diff
@@ -23,7 +23,7 @@ SUBJECT=${2:-screen}
 FILE=${3:-$(xdg-user-dir PICTURES)/$(date +'Grimshot %Y-%m-%d %H-%M-%S.png')}
 if [ "$ACTION" = "usage" ] ; then
   echo "Usage:"
-  echo "  grimshot copy|save win|screen|area [FILE]"
```
Comment (emersion) [inline contrib/grimshot]: Can we name this `output` instead?

**Comment by ohk2kt3t4** [comment]
Would this still capture more than one screen in a multiseat sway session?

**Comment by cyrinux** [comment]
> Would this still capture more than one screen in a multiseat sway session?

I don't know sorry ...

File: contrib/grimshot (hunk 1 above)
Comment (cyrinux) [reply contrib/grimshot]: Yeah good idea, i changed it.

**Comment by ohk2kt3t4** [comment]
> I don't know sorry ...

Was just asking because if that was the case the description might be better as
 "This adds the possibility to capture only the focused screen(s) if several are used"

**Comment by emersion** [comment]
Thanks!

**Pull request merged**